INSERT INTO public.transaction_history (id, cause, created_at, exception, status, transaction_name) VALUES (115, null, '2021-02-06 21:07:36.645246', null, 'SUCCESS', 'LOGIN');
INSERT INTO public.transaction_history (id, cause, created_at, exception, status, transaction_name) VALUES (116, 'This username juanperalta has really exist', '2021-02-06 21:08:18.140772', 'com.tenpo.app.exceptions.UserHasAlreadyExistException', 'FAILED', 'SIGNUP');
INSERT INTO public.transaction_history (id, cause, created_at, exception, status, transaction_name) VALUES (117, null, '2021-02-06 21:08:46.667390', null, 'SUCCESS', 'HISTORY');